### DISCUSSION TITLE
TypeError: Attempted to assign to readonly property. when i import vueflow

### DISCUSSION THREAD

**Discussion by kumareshind** [bcakmakoglu/vue-flow#1137] [General]
hi folks, 

I am getting a strange error when I import 
import { VueFlow } from "@vue-flow/core";

The console error in my browser is 
TypeError: Attempted to assign to readonly property

function oo
vue-flow-core.mjs

Env:

Vue3
Vite

I am not sure if I am missing anything in the vite config.

Could someone help me ?

**Comment by bcakmakoglu** [comment]
Can you provide a little bit more code? Just from this snippet it's kinda impossible to tell what's going on 😄
